thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -217,73 +218,123 @@ public Response computeSeverity(@NotNull @QueryParam("collection") String collec
    * Regenerates anomalies for each window separately
    *
    * As the anomaly result regeneration is a heavy job, we move the function from Dashboard to worker
-   * @param id anomaly function id
+   * @param ids a string containing multiple anomaly function ids, separated by comma (e.g. f1,f2,f3)
```

Comment:
The reason why I need a bulk replay is that when the replay is done, we would want to send the replay result to user. If we send an alert for each function the user creates, it spams the user. Therefore, I need a bulk replay method.